dev/SapphireTest.php

```diff
@@ -496,10 +496,6 @@ public function tearDown() {
 		// Preserve memory settings
 		ini_set('memory_limit', ($this->originalMemoryLimit) ? $this->originalMemoryLimit : -1);
 
-		// Restore email configuration
```

Comment:
Do you think this is the bit that was breaking your tests? Having a null mailer / originalMailer?